gradle/build.gradle

```diff
@@ -0,0 +1,6 @@
+tasks.withType(PublishToMavenRepository) {
+    enabled = false
+}
+tasks.withType(PublishToMavenLocal) {
```

Comment:
Why is this being disabled?

Reply:
Let me look into this, this piece of code was found in multiple locations and I might have missed this one.

Reply:
I think I added this way back because the gradle folder, being a folder/module with submodules, shouldn't itself be published. I asked Remco to double check this (see my comment 2 days ago) since this was added way back.

Reply:
Okay, that makes sense!